theme-frontend-venia/webpack.config.js

```diff
@@ -66,7 +66,7 @@ module.exports = async function(env) {
                     ]
                 },
                 {
-                    test: /\.svg$/,
+                    test: /\.(jpg|svg)$/,
```

Comment:
I know this will be going away, but I'm worried about having `jpg` set to `file-loader` at all, because it'll make it too easy for people cloning/forking to start embedding images inline in their JS.

Can we switch to using inline syntax in `Category/images/index.js`? Example:

```js

export { default as image001 } from 'file-loader!./001.jpg';
export { default as image002 } from 'file-loader!./002.jpg';
export { default as image003 } from 'file-loader!./003.jpg';
```

Reply:
Yeah, I agree. Inline syntax could work here.